src/org/labkey/targetedms/parser/SkylineDocumentParser.java

```diff
@@ -1676,9 +1676,54 @@ else if (XmlUtil.isStartElement(reader, evtType, NOTE))
 
         _precursorCount++;
 
+        updatePrecursorChromInfos(moleculePrecursor);
+
         return moleculePrecursor;
     }
 
+    private <T extends GeneralTransition> void updatePrecursorChromInfos(GeneralPrecursor<T> precursor)
```

Comment:
Does this need to return the transitions? It looks like they're not used.

The method name is kind of ambiguous. How about updateChromatogramSettings() or something like that, and/or a comment about what it's doing?

Reply:
The method does not return anything. But the type parameter is confusing and can be replaced with a wildcard.